pyproject.toml

```diff
@@ -81,6 +81,7 @@ dev = [
     "pytest-check",
     "pytest-mock",
     "pytest-subprocess",
+    "keyring",
```

Comment:
This is the opposite of the PR summary too.  

And why add it for this PR?  We follow conventional commits.  PRs should be narrow and focused on doing one thing. Updating build deps shouldn't be done here.